app/views/components/_markdown-toolbar.html.erb

```diff
@@ -0,0 +1,71 @@
+<markdown-toolbar class="app-c-markdown-toolbar" for="<%= html_for %>">
+  <%= render "govuk_publishing_components/components/details", {
+      title: "Headings and sub-headings"
+  } do %>
+  <ul class="app-c-markdown-toolbar__list">
+    <li class="app-c-markdown-toolbar__item">
+      <md-header-2 class="app-c-markdown-toolbar__button">Heading 2</md-header-2>
+    </li>
+    <li class="app-c-markdown-toolbar__item">
+      <md-header-3 class="app-c-markdown-toolbar__button">Heading 3</md-header-3>
+    </li>
+    <li class="app-c-markdown-toolbar__item">
+      <md-header-4 class="app-c-markdown-toolbar__button">Heading 4</md-header-4>
+    </li>
+  </ul>
+  <a href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk/markdown#headings-and-sub-headings" class="govuk-link" target="_blank"> Guidance on headings</a>
+  <% end %>
+
+  <%= render "govuk_publishing_components/components/details", {
+      title: "Links"
+  } do %>
+  <ul class="app-c-markdown-toolbar__list">
+    <li class="app-c-markdown-toolbar__item">
+      <md-link class="app-c-markdown-toolbar__button">Link</md-link>
+    </li>
+    <li class="app-c-markdown-toolbar__item">
+      <md-link-email class="app-c-markdown-toolbar__button">Email link</md-link-email>
+    </li>
+    <li class="app-c-markdown-toolbar__item">
+      <md-link-cta class="app-c-markdown-toolbar__button">Call to action</md-link-cta>
```

Comment:
I feel like we shouldn't add this feature until it works end-to-end - if it's in the toolbar then I'd expect it to work in preview mode, otherwise it's liable to make the user feel like they did something wrong. Can we leave it out of this PR and add it separately?